File: 프로그래머스/lv0/120897. 약수 구하기/약수 구하기.java
import java.util.*;
import java.util.function.IntFunction;
import java.util.function.IntPredicate;
import java.util.function.Predicate;
import java.util.function.ToIntFunction;
class Solution {
     public int[] solution(int n) {

        ArrayList<Integer> list = new ArrayList<>();


        for (int i = 1; i <=n; i++) {
            if (n % i == 0) {
                list.add(i);
            }
        }
        int[] answer = list.stream().mapToInt(new ToIntFunction<Integer>() {
            @Override
            public int applyAsInt(Integer value) {
                return value;
            }
        }).toArray();
        return answer;
    }
}

File: 프로그래머스/lv0/120897. 약수 구하기/약수 구하기.cs
using System;
using System.Collections.Generic;
public class Solution {
      public int[] solution(int n)
            {
              
                List<int> array = new List<int>();
                int a = 0;
                for (int i = 1; i <= n; i++)
                {
                    if (n % i == 0)
                        array.Insert(a++, i);
                }
                return array.ToArray();
            }
}